src/Faker/Container/ContainerBuilder.php

```diff
@@ -59,8 +59,19 @@ public static function defaultExtensions(): array
         ];
     }
 
+    public static function withDefaultExtensions(): self
+    {
+        $instance = new self();
+
+        foreach (self::defaultExtensions() as $id => $definition) {
+            $instance->add($id, $definition);
+        }
+
+        return $instance;
+    }
```

Comment:
This method can provide an extension point for people who want to

- add extensions
- override default extensions

```php
<?php

declare(strict_types=1);

use Faker\Container;
use Faker\Extension;
use Faker\Generator;

require_once __DIR__ . '/../vendor/autoload.php';

$container = Container\ContainerBuilder::withDefaultExtensions()
    ->add(Extension\FileExtension::class, static function (): Extension\FileExtension {
        return new CustomFileExtension();
    })
    ->build();

$faker = new Generator($container);
```

This method is similar to the method simplified in #720, but returns the `ContainerBuilder` instead of the built `Container`.